include/musica/micm.hpp

```diff
@@ -140,11 +142,21 @@ namespace musica
     /// @param error Error struct to indicate success or failure
     void CreateRosenbrock(const std::string &config_path, Error *error);
 
+    /// @brief Create a BackwardEuler solver of vector-ordered matrix type by reading and parsing configuration file
+    /// @param config_path Path to configuration file or directory containing configuration file
+    /// @param error Error struct to indicate success or failure
+    void CreateBackwardEuler(const std::string &config_path, Error *error);
+
     /// @brief Create a Rosenbrock solver of standard-ordered matrix type by reading and parsing configuration file
     /// @param config_path Path to configuration file or directory containing configuration file
     /// @param error Error struct to indicate success or failure
     void CreateRosenbrockStandardOrder(const std::string &config_path, Error *error);
 
+    /// @brief Create a BackwardEuler solver of standard-ordered matrix type by reading and parsing configuration file
+    /// @param config_path Path to configuration file or directory containing configuration file
+    /// @param error Error struct to indicate success or failure
+    void CreateBackwardEulerStandardOrder(const std::string &config_path, Error *error);
+
```

Comment:
I have the same thoughts on the ordering of the functions related to the above comments.